routes/utils.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function(connection) {
```

Comment:
Is this common practice to put a utility alongside routes? Since this isn't a route, I'd consider moving the `utils.js` file to the parent folder. I would expect a file at `routes/utils.js` to contain an express script to handle HTTP requests to `/utils`

Reply:
I put it in there because it was related to routes, but now that I think of it, it makes less sense to have it there. Moved it the parent folder. :)